### PR TITLE
Manifestv2

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

**Comment by autra** [comment]
I think change_url has never been supported upstream, we can disable it.

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+userdoc = "https://wiki.diasporafoundation.org/FAQ_for_users"
+code = "https://github.com/diaspora/diaspora"
+
+[integration]
+yunohost = ">=11.2"
+architectures = "all" # FIXME: can be replaced by a list of supported archs using the dpkg --print-architecture nomenclature (amd64/i386/armhf/arm64), for example: ["amd64", "i386"]
+multi_instance = true
+ldap = false
+sso = false
+disk = "50M" # FIXME: replace with an **estimate** minimum disk requirement. e.g. 20M, 400M, 1G, ...
```
Comment (autra) [inline manifest.toml]: 419M for my actual `/var/www/diaspora` without uploads, so maybe a bit more for ruby and other deps if we count them? 500M I'd say.

**Comment by autra** [comment]
I would have preferred you wait for me to read this...

**Comment by Salamandar** [comment]
> I think change_url has never been supported upstream, we can disable it.

I added change_url (for domain only), as the domain is only stored in the config file :)

File: manifest.toml (hunk 1 above)
Comment (Salamandar) [reply manifest.toml]: Yeah, indeed. Fixed in master directly.

**Pull request merged**